draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -931,6 +933,73 @@ As noted in {{multiplexing}}, all RoQ streams and RoQ datagrams begin with a flo
 
 Because QUIC provides a capability to migrate connections for various reasons, including recovering from a path failure ({{Section 9 of !RFC9000}}), when a QUIC connection migrates, a RoQ sender has the opportunity to revisit decisions about which RTP packets are encapsulated in QUIC streams, and which RTP packets are encapsulated in QUIC DATAGRAMs. Again, RoQ receivers need to be prepared for this eventuality.
 
+## RTCP Considerations
+
+RTCP was originally defined to be used with UDP, which implies (1) that
+transmission timing is largely under the control of the application (limited
+buffering inside the stack) and (2) that the overhead, *avg_rtcp_size*, used to
+compute the RTCP transmission interval could be deterministically computed by
+adding the IP asnd UDP headers. Both change when carrying RTCP over QUIC and
+they change in different ways when using QUIC streams vs. QUIC datagrams.
+
+### RTCP over QUIC datagrams {#rtcp-over-datagrams}
+
+When sending RTCP packets in QUIC datagrams this implies that a packet may not
+be immediately transmitted as it is subject to queuing and multiplexing with RTP
+packets and subject to congestion control. This means that a sending timestamp
+added to an RTCP packet, e.g., in an SR packet, may differ in unforeseeable ways
+from the actual time when the packet gets sent into the network while these are
+usually fairly close to each other for RTP-over-UDP. Effectively, we have a
+application sending timestamp *t_a* and the network transmission timestamp
+*t_n*. Applications just have to be aware that RTCP does not measure the network
+level RTT but rather the application layer RTT.
+
+Moreover, the true overhead per RTCP packet cannot easily be determined: this is
+because, in addition to adding the IP and UDP headers, the QUIC (short) header
+and the QUIC datagram frame header are to be considered but their sizes vary and
+it is unknown which other frames may be sent along in the same UDP packet. Any
+lower bound that can be determined could be affected by the version of QUIC
+being used.
+
+It is thus suggested that applications do _not_ attempt to include lower layer
+headers but only work with the net data rate of the codec including the RTP
+headers and of the RTCP packets, both excluding QUIC, UDP, and IP header
+overhead.
```

Comment:
```suggestion
It is thus suggested that application developers recognize that per-RTCP packet overhead will always be an estimate, and include IP, UDP, QUIC, and DATAGRAM header sizes as a conservative heuristic.
```
Stolen from @gloinul - thank you, Magnus!

Reply:
I think this should also include QUIC STREAM frames.

Reply:
oh, no, sorry, STREAMs are covered below...